src/components/html/EcAppsMarket.html

```diff
@@ -34,28 +28,29 @@ <h2>{{ i19applications }}</h2>
     v-if="!loading" 
     class="pt-2"
   >
-    <div
-      class="card bg-pale-yellow border p-2"
+    <div 
+      class="card bg-pale-yellow border p-2" 
       v-if="loadError"
-      key="error"
     >
       {{ errorMessage }}
       <div>
         <a 
-          role="button"
+          role="button" 
           @click="updateTabContent"
         >
           {{ i19tryAgain }}
         </a>
       </div>
     </div>
 
-    <article v-else key="list">
+    <article 
+      v-else 
+      key="list"
+    >
       <div class="row">
-        <div
-          class="col-6 col-md-4 col-lg-3"
+        <div 
+          class="col-6 col-md-4 col-lg-3" 
           v-for="(app, i) in apps"
-          :key="`app-${i}`"
```

Comment:
Esta `key` aqui não poderíamos tirar não, em loop (`v-for`) sempre deve haver `key` em todos os elementos, se não o Vue se confunde de vez em quando...